python/tests/test_v2_row.py

```diff
@@ -36,6 +36,18 @@ def test_stachv2_add_table(self):
             self.assertEqual(headerColumns, tables[0].columns.values.tolist())
             self.assertEqual(firstRow, tables[0].iloc[0].values.tolist())
 
+    def test_stachv2_metadata(self):
+        file = os.path.join(ROOT_DIR, "resources", "V2RowStachData.json")
+        with open(file) as f:
+            data = json.load(f)
+            table = data['tables']['3805a4c8-5493-4e63-9d71-c553d20f266b']
+            stachBuilder = StachExtensionFactory.get_row_organized_builder(StachVersion.V2)
+            stachExtension = stachBuilder.add_table("id1", table).build()
```

Comment:
Can you use the same code as test_v2_simplifiedrow.test_stachv2_metadata

Reply:
done https://github.com/factset/stach-extensions/pull/22/commits/2e3c1785f2727de517f932a0cd995b8b924b18a8